src/views/LimitedRestorationExtension.vue

```diff
@@ -204,18 +213,20 @@ export default class LimitedRestorationExtension extends Mixins(
     // do not proceed if app is not ready
     if (!val) return
 
-    // do not proceed if FF is disabled
-    // bypass this when Vitest is running as FF are not fetched
-    if (!this.isVitestRunning && !GetFeatureFlag('restoration-ui-enabled')) {
-      window.alert('Restorations are not available at the moment. Please check again later.')
+    // do not proceed if we are not authenticated (safety check - should never happen)
+    if (!this.isAuthenticated) return
```

Comment:
I added this safety check, and below, I moved the auth check before the FF check.